src/test/java/com/google/javascript/clutz/partial/global_alias_missing.js

```diff
@@ -0,0 +1,8 @@
+goog.provide('ns.event');
+
+//!! Assume there exists a ns.event.EventTarget, then
+//!! emitting just t: EventTarget would be wrong.
+//!! TypeScript will pick up ns.event.EventTarget instead
+//!! of the global EventTarget.
+/** @param {EventTarget} t */
```

Comment:
I still don't understand.  Why doesn't `EventTarget` refer to the local value?

Could you add a contrasting function that does refer to `ns.event.EventTarget`?  Or at least show how the local value would be defined?

Reply:
Good idea, added f2 showing how namespace local Event target would look like